packages/web-vite/src/lib/graphql-client.ts

```
@@ -3,13 +3,14 @@
 
 import { useCallback, useState } from 'react'
 
-import type { DeleteResult, HighlightColor, LibraryItem } from '../types/api'
+import type { DeleteResult, HighlightColor, LibraryItem, RssFeed } from '../types/api'
```

ESLint

[error] Replace `·DeleteResult,·HighlightColor,·LibraryItem,·RssFeed·` with `⏎··DeleteResult,⏎··HighlightColor,⏎··LibraryItem,⏎··RssFeed,⏎`

(prettier/prettier)